runtime/lua/vim/lsp/codelens.lua

```diff
@@ -172,7 +172,7 @@ local function display_line_lenses(bufnr, ns, line, lenses)
   end
 
   api.nvim_buf_clear_namespace(bufnr, ns, line, line + 1)
-  if #chunks > 0 then
+  if #chunks > 0 and line <= api.nvim_buf_line_count(bufnr) then
```

Comment:
Can't we check that `line <= api.nvim_buf_line_count(bufnr)` before computing the chunks?

Reply:
moved the check to the call site